python/cudaq/dbg/__init__.py

```diff
@@ -6,4 +6,4 @@
 # the terms of the Apache License 2.0 which accompanies this distribution.     #
 # ============================================================================ #
 
-from .ast import print_i64, print_f64
+from .ast import print_i1, print_i64, print_f64
```

Comment:
Is it possible to have a `kernel_print` function that takes a descriptor argument (like C's `printf`) instead of having all these different functions with the type as part of the name?